### PR TITLE
CMS: templates: correctionto research guide

### PR DESCRIPTION
**Pull request by katilp**
This addresses #1994, but does not address corrections depending on #1888,  #1996, #1956

### CODE REVIEW

File: cernopendata/modules/fixtures/data/docs/cms-guide-for-research/cms-guide-for-research.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ However, if you are interested in finding hints, tips and guidance for conductin
 
 > "I want to learn about the terms under which I can access and use the CMS Open Data, and publish results obtained from them."
 
-- Go to ["Data preservation and open access policy"](/record/411) (if you are a CMS member see also ["Rules for use of open access CMS data by individual members of CMS"](https://cms-docdb.cern.ch/cgi-bin/DocDB/ShowDocument?docid=12242)).
+- Go to ["Data preservation and open access policy"](/record/411) (if you are a CMS member see also an internal document ["Rules for use of open access CMS data by individual members of CMS"](https://cms-docdb.cern.ch/cgi-bin/DocDB/ShowDocument?docid=12242)).
```
Comment (RaoOfPhysics) [inline cernopendata/modules/fixtures/data/docs/cms-guide-for-research/cms-guide-for-research.md]: Can we change this to: "if you are a CMS member, also see the internal document…"?

File: cernopendata/modules/fixtures/data/docs/cms-guide-for-research/cms-guide-for-research.md [hunk 2]
```diff
@@ -170,14 +170,14 @@ However, if you are interested in finding hints, tips and guidance for conductin
...
-> "I want to use the public analysis example from the MIT jet-analysis group's papers."
+> "I want to learn about the published jet-analysis papers by the MIT group."
 
 - See *Jet Substructure Studies with CMS Open Data*, A. Tripathee et al., Apr 19, 2017, MIT-CTP-4890, [arXiv:1704.05842](https://inspirehep.net/record/1593756),
 - and *Exposing the QCD Splitting Function with CMS Open Data*, A. Larkoski et al., Apr 17, 2017, MIT-CTP-4891, [arXiv:1704.05066](https://inspirehep.net/record/1591972).
 
 <center>–––</center>
 
-> "I want to (re)validate the 2010 datasets within my setup…"
+> "I want to run the examples used for 2010 datasets validation within my setup…"
```
Comment (RaoOfPhysics) [inline cernopendata/modules/fixtures/data/docs/cms-guide-for-research/cms-guide-for-research.md]: "I want to run the examples used for validation of the 2010 datasets within my setup…"

File: cernopendata/modules/fixtures/data/docs/cms-guide-for-research/cms-guide-for-research.md (hunk 1 above)
Comment (katilp) [reply cernopendata/modules/fixtures/data/docs/cms-guide-for-research/cms-guide-for-research.md]: 👍

**Comment by katilp** [comment]
Thanks, done (and one additional FIXME for trigger example record, and removing "to be added to the portal")

**Pull request merged**